nixos/modules/services/misc/atuin.nix

```diff
@@ -0,0 +1,87 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.atuin;
+in
+{
+  options = {
+    services.atuin = {
+      enable = mkEnableOption (mdDoc "Enable server for shell history sync with atuin.");
+
+      openRegistration = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
Not sure about this. You want to turn it off after registering your accounts to prevent randoms from loading their history into your instance.

Reply:
Agree -- switched default to `false` to match upstream defaults.